tests/functional/testOnboarding.js

```diff
@@ -369,4 +391,376 @@ describe('Onboarding', function() {
+      const blockAdsDisabledEvents = await vpn.gleanTestGetValue("interaction", "blockAdsDisabled", "main");
+      assert.equal(blockAdsDisabledEvents.length, 1);
+      const blockAdsDisabledEventsExtras = blockAdsDisabledEvents[0].extra;
+      assert.equal(privacyScreenTelemetryId, blockAdsDisabledEventsExtras.screen);
+
+      //Verify that blockTrackersEnabled event is recorded
+      await vpn.waitForQueryAndClick(queries.screenOnboarding.PRIVACY_BLOCK_TRACKERS_CHECKBOX.visible());
+      const blockTrackersEnabledEvents = await vpn.gleanTestGetValue("interaction", "blockTrackersEnabled", "main");
+      assert.equal(blockTrackersEnabledEvents.length, 1);
+      const blockTrackersEnabledEventsExtras = blockTrackersEnabledEvents[0].extra;
+      assert.equal(privacyScreenTelemetryId, blockTrackersEnabledEventsExtras.screen);
+
+      //Verify that blockTrackersDisabled event is recorded
+      await vpn.waitForQueryAndClick(queries.screenOnboarding.PRIVACY_BLOCK_TRACKERS_CHECKBOX.visible());
+      const blockTrackersDisabledEvents = await vpn.gleanTestGetValue("interaction", "blockTrackersDisabled", "main");
+      assert.equal(blockTrackersDisabledEvents.length, 1);
+      const blockTrackersDisabledEventsExtras = blockTrackersDisabledEvents[0].extra;
+      assert.equal(privacyScreenTelemetryId, blockTrackersDisabledEventsExtras.screen);
+
+      //Verify that blockMalwareEnabled event is recorded
+      await vpn.waitForQueryAndClick(queries.screenOnboarding.PRIVACY_BLOCK_MALWARE_CHECKBOX.visible());
+      const blockMalwareEnabledEvents = await vpn.gleanTestGetValue("interaction", "blockMalwareEnabled", "main");
+      assert.equal(blockMalwareEnabledEvents.length, 1);
+      const blockMalwareEnabledEventsExtras = blockMalwareEnabledEvents[0].extra;
+      assert.equal(privacyScreenTelemetryId, blockMalwareEnabledEventsExtras.screen);
+
+      //Verify that blockMalwareDisabled event is recorded
+      await vpn.waitForQueryAndClick(queries.screenOnboarding.PRIVACY_BLOCK_MALWARE_CHECKBOX.visible());
+      const blockMalwareDisabledEvents = await vpn.gleanTestGetValue("interaction", "blockMalwareDisabled", "main");
+      assert.equal(blockMalwareDisabledEvents.length, 1);
+      const blockMalwareDisabledEventsExtras = blockMalwareDisabledEvents[0].extra;
+      assert.equal(privacyScreenTelemetryId, blockMalwareDisabledEventsExtras.screen);
+
+      //Verify that goBackSelected event is recorded
+      await vpn.waitForQueryAndClick(queries.screenOnboarding.PRIVACY_BACK_BUTTON.visible());
+      const goBackSelectedEvents = await vpn.gleanTestGetValue("interaction", "goBackSelected", "main");
+      assert.equal(goBackSelectedEvents.length, 1);
+      const goBackSelectedEventsExtras = goBackSelectedEvents[0].extra;
+      assert.equal(privacyScreenTelemetryId, goBackSelectedEventsExtras.screen);
+
+      await advanceToSlide(1)
+      await vpn.gleanTestReset();
```

Comment:
the only time `gleanTestReset` is being used is to reset the count of `continueSelected` events right before we record them in onboarding because `continueSelected` is also recorded during authentication. Breaking up these won't resolve this issue

I will, however, add a comment